### PR TITLE
[DROOLS-1446] ruleunit implementation

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: kie-api/src/main/java/org/kie/api/KieServices.java [hunk 1]
```diff
@@ -316,9 +316,16 @@
...
+     */
+    static KieServices get() {
+        return Factory.get();
+    }
+
+    /**
      * A Factory for this KieServices
      */
-    public static class Factory {
+    class Factory {
```
Comment (tarilabs) [inline kie-api/src/main/java/org/kie/api/KieServices.java]: While I think this is not related to this PR and I welcome the added method above, I would like to have double-confirmation this is a (/another) breaking change in the Kie API, correct?

Comment (mariofusco) [inline kie-api/src/main/java/org/kie/api/KieServices.java]: No, breaking change: public static are just redundant in an interface. The class is public and static by default.

Comment (tarilabs) [inline kie-api/src/main/java/org/kie/api/KieServices.java]: My whiteboard compiler needs some serious tuning and more :coffee: 
Thanks.
